crowbar_framework/lib/crowbar/repository.rb

```diff
@@ -203,18 +215,32 @@ def provided_with_enabled?(feature, platform = nil, arch = nil, check_enabled =
             found = true
 
             r = new(platform, arch, repo)
+
             answer &&= r.available?
+            unless answer
```

Comment:
Isn't the check wrong? Shouldn't we just test on `r.available?` here (as if a previous repo is available, we won't fill the missing repos at all).